Reqnroll/Reqnroll.nuspec

```diff
@@ -36,7 +36,6 @@
     <file src="bin\$config$\netstandard2.0\Reqnroll.dll" target="lib\netstandard2.0" />
     <file src="bin\$config$\netstandard2.0\Reqnroll.pdb" target="lib\netstandard2.0" />
     <file src="bin\$config$\netstandard2.0\Reqnroll.xml" target="lib\netstandard2.0" />
-    <file src="bin\$config$\netstandard2.0\Reqnroll.BoDi.*" target="lib\netstandard2.0" />
     <file src="bin\$config$\netstandard2.0\Reqnroll.Utils.*" target="lib\netstandard2.0" />
```

Comment:
could we also remove this one? As I don't see it in the package of v2?

<img width="423" height="208" alt="Image" src="https://github.com/user-attachments/assets/e8a3df2f-3530-46d8-b524-eb2e646710f7" />

Reply:
ah I should have checked v3 :)

<img width="768" height="217" alt="image" src="https://github.com/user-attachments/assets/5cf65e5c-037a-4993-94b9-0fef89d8fa3d" />